### DISCUSSION TITLE
VM image directory

### DISCUSSION THREAD

**Discussion by graiendor** [abiosoft/colima#1162] [Q&A]
Hello!
I want to install colima on a host without network.
Colima start command tries to download vm image and fails. Where do I manually put this vm image to make it work?

**Comment by abiosoft** [comment]
The initial start of Colima requires internet and there is no workaround yet. 
Do you want to manually download the Colima disk image or you want to be able to use your custom disk image?

**Reply by graiendor** [reply]
Sad, thank you.
I want to download vm image the Colima tries to download at start, but I won't have network on this host either way.
This one:
ubuntu-24.04-minimal-cloudimg-arm64-containerd.qcow2

**Reply by abiosoft** [reply]
It is a feature being considered.
